lib/engine/game/g_18_esp/entities.rb

```diff
@@ -119,7 +120,8 @@ module Entities
             name: 'Ferrocarril Vasco-Navarro',
             value: 160,
             revenue: 20,
-            desc: 'It provides a 10% certificate from the Southern company CRB.',
+            min_price: 1,
```

Comment:
This isn't needed. There are 2 P6 companies, one picks at random, one picks the CRB. Depends on which companies exist in the game.

see game.setup_companies for the setup logic

Reply:
Ah yeah I see it now. Interesting approach. Fixed.